src/speculators/models/dflash2/core.py

```
@@ -0,0 +1,195 @@
+    "DFlash2DraftModel",
+]
+
+_DEFAULT_LOSS_CONFIG: LossConfig = {"kl_div": (kl_div_loss, 1.0)}
+
+
+@SpeculatorModel.register("dflash2")
+class DFlash2DraftModel(DFlashDraftModel):
+    """DFlash with local convolution and bilinear candidate reranking."""
+
+    config_class: ClassVar[type[DFlash2SpeculatorConfig]] = DFlash2SpeculatorConfig  # type: ignore[misc,assignment]
+    _no_split_modules = ["Qwen3DFlash2DecoderLayer"]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)